static/app/views/insights/common/components/widgets/overviewSlowAssetsWidget.tsx

```diff
@@ -0,0 +1,191 @@
+    {
+      fields: [
+        'span.group',
+        'project.id',
+        'sentry.normalized_description',
+        totalTimeField,
+      ],
+      sorts: [{field: totalTimeField, kind: 'desc'}],
+      search,
+      limit: 3,
+      noPagination: true,
+    },
+    referrer
+  );
+
+  const {
+    data: assetSeriesData,
+    isLoading: isAssetSeriesLoading,
+    error: assetSeriesError,
+  } = useTopNSpanSeries(
+    {
+      search: `span.group:[${assetListData?.map(item => `"${item['span.group']}"`).join(',')}]`,
+      fields: [groupBy, yAxes],
+      yAxis: [yAxes],
+      sort: {field: yAxes, kind: 'desc'},
+      topN: 3,
+      enabled: assetListData?.length > 0,
+    },
+    referrer
+  );
+
+  const isLoading = isAssetSeriesLoading || isAssetListLoading;
+  const error = assetSeriesError || assetListError;
+
+  const hasData = assetListData && assetListData.length > 0 && assetSeriesData.length > 0;
+
+  const colorPalette = theme.chart.getColorPalette(assetSeriesData.length - 1);
+
+  const visualization = (
+    <WidgetVisualizationStates
+      isEmpty={!hasData}
+      isLoading={isLoading}
+      error={error}
+      emptyMessage={<WidgetEmptyStateWarning />}
+      VisualizationType={TimeSeriesWidgetVisualization}
+      visualizationProps={{
+        id: 'overviewSlowAssetsWidget',
+        showLegend: props.loaderSource === 'releases-drawer' ? 'auto' : 'never',
+        plottables: assetSeriesData.map(
+          (ts, index) =>
+            new Line(convertSeriesToTimeseries(ts), {
+              color: colorPalette[index],
+            })
+        ),
+        ...props,
+        ...releaseBubbleProps,
+      }}
+    />
+  );
+
+  const footer = hasData && (
+    <WidgetFooterTable>
+      {assetListData?.map((item, index) => (
+        <Fragment
+          key={`${item['project.id']}-${item['span.group']}-${item['sentry.normalized_description']}`}
+        >
+          <div>
+            <SeriesColorIndicator
+              style={{
+                backgroundColor: colorPalette[index],
+              }}
+            />
+          </div>
+          <div>
+            <SpanDescriptionCell
+              projectId={Number(item['project.id'])}
+              group={item['span.group']}
+              description={item['sentry.normalized_description']}
+              moduleName={ModuleName.RESOURCE}
+            />
+          </div>
+          <span>{getDuration((item[totalTimeField] ?? 0) / 1000, 2, true)}</span>
+        </Fragment>
+      ))}
+    </WidgetFooterTable>
+  );
+
+  const exploreUrl = getExploreUrl({
+    selection,
+    organization,
+    visualize: [
+      {
+        chartType: ChartType.LINE,
+        yAxes: [yAxes],
+      },
+    ],
+    mode: Mode.AGGREGATE,
+    title: t('Assets by Time Spent'),
+    query: search?.formatString(),
+    sort: undefined,
+    groupBy: [groupBy],
+    referrer,
+  });
+
+  const chartActions = (
+    <BaseChartActionDropdown
+      key="http response chart widget"
```

Comment:
This should probably change to something like


```suggestion
      key="slow assets widget"
```